README.md

```diff
@@ -1 +1,20 @@
-Project-Portfolio-setup-and-mobile-version-skeleton
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project : Setup And Mobile Version Skeleton
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```

Comment:
Please add a description of the project features here. Moreover, please add steps to run the project locally. README should be professional. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse.